gluecodium/src/main/resources/templates/jni/utils/JniExceptionThrowerHeader.mustache

```diff
@@ -30,7 +30,7 @@ namespace {{.}}
 {{/internalNamespace}}
 namespace jni
 {
-    class JniExceptionThrower
+    class JNIEXPORT JniExceptionThrower
```

Comment:
This class is header only and shouldn't be exported

Reply:
Done